### DISCUSSION TITLE
BT-712(a)-LotResult and review-type.gc

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#1027] [Ideas]
We have noticed that the field BT-712(a)-LotResult with the corresponding code list review-type.gc does not provide any value. Since complainants are the only possible code. If there are complainants, I could simply enter the number in field BT-712(b)-LotResult.

Are there plans to add more codes to the codelist? Otherwise, we would suggest merging the fields BT-712(a)-LotResult and BT-712(b)-LotResult and only asking for the number of complainants.

**Comment by YvesJo** [comment]
Hi,
The list was initially populated with 2 different codes before getting reduced to a single one. There is no indication that the list won't get updated in the future. 
The structure is also reused and requires, in addition to the number, a code to specify what the number is about.
Another option would be to define a very specific XML element which would restrict future evolution and, in case of change, would require a schema update.

**Comment by JanEmde** [comment]
Understood. But then you should at least change the name (incl. description fields) of the field BT-712(a)-LotResult. Currently, both fields are called ‘Number of complainants’.

**Reply by YvesJo** [reply]
Hi,
I fully agree with the suggestion, The 2 fields actually have different names but a same label associated which is not expected and should be addressed with further labels fixes.
